mb-solr/pom.xml

```diff
@@ -129,6 +129,24 @@
 
 	<build>
 		<plugins>
+			<plugin>
+			  <groupId>de.jflex</groupId>
```

Comment:
Nitpicking: the indentation here mixes tabs and spaces.

Reply:
Will fix.